src/main/deploy/pathplanner/autos/4 pieces L4 blue.auto

```diff
@@ -0,0 +1,91 @@
+{
```

Comment:
ℹ️ I am not gonna look at any of these path files, but I would prefer they be named more cleanly e.g. no spaces in filenames.